include/sqlpp11/detail/type_set.h

```diff
@@ -246,6 +246,21 @@ namespace sqlpp
 
     template <template <typename> class Transformation, typename T>
     using transform_set_t = typename transform_set<Transformation, T>::type;
+	
+	template <typename T>
+	struct make_name_of_set
```

Comment:
Does MSVC have a beef with `transform_set_t<name_of, _unique_tables>`?
Or do you want to introduce a short hand notation?

In the former case, again, please add a reference to the MSCV issue.
In the latter case, please remove it from this pull request which is about MSVC only (and I have to admit, I am not convinced that this notation is easier to comprehend).